File: RSPAClient/Atomic/app/Entities/MT_CYCT_EVENT_HDR.ts
export class MT_CYCT_EVENT_HDR {
    public BUSINESS_UNIT: string;
    public EVENT_ID: string;
    public USER_ID: string;
    public UPDATE_DATE?: Date;
    public TRANSACTION_ID: number;
    public COMPLETED_DATE?: Date;
    public PARENT_EVENT_ID: string;
    public EVENT_STATUS?: number;
    public UPDATE_USER_ID: string;
    public SEND_DATE_TIME?: Date;
    public EVENT_TYPE: number;
}

File: RSPAClient/Core/AtPar.POCOEntities/MT_CYCT_EVENT_HDR.cs
using System;
using System.Collections.Generic;

namespace AtPar.POCOEntities
{
    public partial class MT_CYCT_EVENT_HDR
    {
        public string BUSINESS_UNIT { get; set; }
        public string EVENT_ID { get; set; }
        public string USER_ID { get; set; }
        public Nullable<System.DateTime> UPDATE_DATE { get; set; }
        public int TRANSACTION_ID { get; set; }
        public Nullable<System.DateTime> COMPLETED_DATE { get; set; }
        public string PARENT_EVENT_ID { get; set; }
        public Nullable<int> EVENT_STATUS { get; set; }
        public string UPDATE_USER_ID { get; set; }
        public Nullable<System.DateTime> SEND_DATE_TIME { get; set; }
        public int EVENT_TYPE { get; set; }
        public string EVENTUSERS { get; set; }
    }
}
